src/domain/event/utils.ts

```diff
@@ -65,6 +65,15 @@ export const getEventStartTimeStr = (
 export const isEventFree = (event: EventFieldsFragment): boolean =>
   Boolean(event.offers.find((item) => item.isFree)?.isFree);
 
+/**
+ * Check if the additional information is requested as mandatory.
+ * @param eventData
+ * @return {boolean}
+ */
```

Comment:
Do these 3 lines add something? these are already documented with typescript. I know these are used somewhere in the codebase but I think these are not needed

Reply:
Because we are coding with TS and have types there, I can agree when the function is as simple as this, or like the other functions in this file. I removed all the manual API-doc comments there - they were just noise.